wiki/en/en-Installation-for-Windows.md

```diff
@@ -46,15 +46,15 @@ Before you start with Jamulus:
 
```

Comment:
In the above (line 44), why does it only mention inputs?   I think this means "Click the plus icon to open the sound card you want to use to show its inputs and outputs, if they are not shown"

Reply:
Done